document/chapters/02-background.tex

```diff
@@ -345,51 +344,50 @@ \section{Ray Tracing Acceleration Data Structures}
-of the parent partition, and bounds of the candidate child partitions.
+another. The second is the evaluation of the ``cost'' of that split. This cost
+is representative of the comparison between a division of the node for several
+candidate split planes and forming a leaf from the current node.  Because there
+is no way to know exactly how expensive or inexpensive the cost of a split will
+be for the particular simulation at hand, heuristics are used to estimate this
+cost and determine the optimal splitting plane using limited information about
+the local nodes in the tree. More specifically, this information is typically
+limited to the number of primitives being split, bounds of the parent partition,
+and bounds of the candidate child partitions.
 
 A virtually infinite number of planes could be tested to find the optimal plane for
-dividing the entities between the two child volumes, but even if one were to
+dividing the entities between the child bounding volumes, but even if one were to
 encounter such a splitting plane, it can be difficult to identify the plane as
 such without more knowledge about the final tree structure. As a result, a
 limited set of planes is tested for the best split based on a set of assumptions
 about the problem at hand and the heuristics being used to evaluate split
 costs. The most common method for split plane candidates is median plane
-splitting in which the current partition is split in half for each axes of the
-current bounding volume. The splitting planes are then evaluated and the split
+splitting in which the current partition is split in half along each axes of the
+current bounding volume. Splitting plane costs are then evaluated and the split
 with the lowest cost is selected. Other methods for plane selection exist, but
-will not be discussed as this work is more focused on traversal performance.
+will not be discussed here as this work is more focused on traversal performance.
 
-Two heuristics will be discussed here - the Entity Ratio Heuristic (ERH) and the
+Two heuristics will now be discussed - the Entity Ratio Heuristic (ERH) and the
 Surface Area Heuristic (SAH). The ERH uses the resulting number of primitives in
 each child node to determine the cost of a split. The philosophy behind this
 heuristic is to maintain the expected $O(log(N_{primitives}))$ cost of a ray
-traversal by ensuring that primitives are split as evenly as possible from
+traversal by ensuring that the number primitives are split as evenly as possible from
 parent to child node. A form of this heuristic is presented in Equation
-\eqref{eq:ERH}. The ERH cost is unitless and bounded by zero and one. This
+\eqref{eq:ERH}. The ERH cost is unit-less and bounded by zero and one. This
 heuristic provides a finite limit on the expected cost, and makes it possible to
-set both an upper and lower bound as both unacceptably high and ``good enough''
+set both an upper and lower bound as both an unacceptably high and ``good enough''
```

Comment:
and a ``good enough''

If you add the article before "unacceptably high" then you need to add an article here too